tests/test_biophysical/test_animal/test_feedlot/test_beef_nrc_calculator.py

```
@@ -0,0 +1,423 @@
+def calc():  # type: ignore
+    """BeefNRCRequirementsCalculator class."""
+    from RUFAS.biophysical.animal.nutrients.beef_nrc_requirements_calculator import (
+        BeefNRCRequirementsCalculator,
+    )
+
+    return BeefNRCRequirementsCalculator
+
+
+@pytest.mark.unit
+def test_calculate_sbw(calc) -> None:  # type: ignore
+    """SBW must be 0.96 × BW."""
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)